CMakeLists.txt

```diff
@@ -43,11 +43,11 @@ ExternalProject_Add(pybind11
 
 option(WERROR "Treat warnings as errors" ON)
 
-function(bito_compile_opts PRODUCT)
+function(bito_compile_opts PRODUCT _WERROR)
```

Comment:
I just added this argument so that the "treat warnings as errors" flag is toggled off for Noodle.

Reply:
I set it up so that when `bito` is compiled, it enforces warnings as errors, but the files in `extras/` does not treat warnings as errors (I felt like since these are just for noodling, it was unnecessary to enforce things like unused variables and such).